extras/paid-courses.md

```diff
@@ -86,6 +86,10 @@
 ### Artificial Intelligence
 
 ### Machine Learning
+Name | Provider
+:-- | :--:
+[Machine Learning by Andrew Ng](https://www.coursera.org/learn/machine-learning) | [Coursera](https://www.coursera.org)
```

Comment:
Yeah, and table's headers should follow the same pattern of the main curriculum and the free-courses file.

```
Courses | Duration | Effort
:-- | :--: | :--:
```